site/docs/v1/tech/apis/users.adoc

```diff
@@ -248,7 +248,11 @@ include::docs/v1/tech/apis/_user-response-body.adoc[]
 
 == Delete a User
 
-This API is used to delete a User. You must specify the Id of the User on the URI. You can also specify whether or not the User is soft or hard deleted. Soft deleted Users are marked as inactive but not deleted from FusionAuth.
+This API is used to delete a User. You must specify the Id of the User on the URI. You can also specify whether or not the User is soft or hard deleted. A soft delete deactivates the User. A hard delete permanently deletes a User's data.
+
+Soft deleted Users are marked as inactive but not deleted from FusionAuth. Deactivated users' data is retained but they are unable to authenticate. Users who have been deactivated can be reactivated; see <<Reactivate a User>> for more.
+
+Hard deleted Users' data is permanently removed from FusionAuth. The User's data cannot be restored via the FusionAuth API or the administrative user interface. If you need to restore the data, retrieve it from a database backup.
```

Comment:
You use `users'` in other spots, here, did you mean to capitalize `Users'`?

Reply:
Thanks, made it more consistent.